src/components/ContentSidebar/DetailsSidebar.js

```diff
@@ -73,13 +71,22 @@ class DetailsSidebar extends React.PureComponent<Props, State> {
     constructor(props: Props) {
         super(props);
         this.state = {
-            file: props.file,
```

Comment:
I actually didn't realize we were tracking the same data in two (or more) different states. That was bound to lead to trouble. Glad we're starting to address it.